### DISCUSSION TITLE
Hengband <3

### DISCUSSION THREAD

**Discussion by yesDaniel** [hengband/hengband#3759] [General]
It's a huge surprise that Hengband, my favorite variant branch, is back and has an English release too!
I was away from @Bands for 10 years and just discovered this great surprise..
Also there are two ocidental branches of Hengband that would be worthy taking a look, lots of interesting things:
FrogComposband and PosChengband!
Well, I installed and will play them all X)
Just wrote the news of Hengband on Reddit community!
Thank you and keep it up! :D

**Comment by deskull-m** [comment]
Thanks to Reddit for making the news! We hope you continue to enjoy it!
